src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Provides an output implementation that sends diagnostics data to Azure Event Hubs.</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>net452;netstandard1.6;netstandard2.0</TargetFrameworks>
```

Comment:
We could also keep producing net452 binary (upgrade the dependency to ver 1.0.3), and refer to version 4.3.1 for netstandard 1.6 and 2.0. .NET 4.5.2 is still supported so maybe it is too early to discontinue that...

Reply:
That's a good point